tools/myst/host/package.c

```diff
@@ -126,6 +129,21 @@ int _package(int argc, const char* argv[])
         goto done;
     }
 
+    cli_getopt(&argc, argv, "--signing-engine-name", &signing_engine_name);
+    cli_getopt(&argc, argv, "--signing-engine-path", &signing_engine_path);
+    cli_getopt(&argc, argv, "--signing-engine-key", &signing_engine_key);
+    // engine path optional
```

Comment:
Is this documented somewhere?

> // engine path optional

Reply:
are you referring to the new parameters, or the optional path parameter?
from oe-sign help: 
  -n, --engine             name of the crypto engine to use, for example
                           "pkcs-11".
  -i, --key-id             engine-specific ID string specifying the
                           desired enclave signing key from the engine.
  -p, --load-path          [optional] absolute path to the shared object
                           that implements the engine.